### PR TITLE
DNSLeak

### PR DESCRIPTION
**Pull request by yoursoftwarebug**
1. Enable "Do not send UDP(allow only local)" checkbox. <--- not exist yet
2. In "advanced", set both "VPN DNS" to 127.0.0.1. <--- bad address error
3. Set DNS port to 9090(Orbot's DNSPort). <--- not exist

Please improve netguard.

### CODE REVIEW

**Comment by licaon-kter** [comment]
What is this exactly?

File: FAQ.md [hunk 1]
```diff
@@ -538,6 +538,11 @@ like having the Tor network unreacheable, your activity actively monitored/targe
 online services (eg. Gmail, Google Play store) failing to login or being forced to solve endless capchas
 when accessing sites that use Cloudflare's CDN services.
 
+Fix DNS leak;
+1. Enable "Do not send UDP(allow only local)" checkbox.
+2. In "advanced", set both "VPN DNS" to 127.0.0.1.
```
Comment (M66B) [inline FAQ.md]: 127.0.0.1 is not a valid DNS server address

File: FAQ.md [hunk 2]
```diff
@@ -538,6 +538,11 @@ like having the Tor network unreacheable, your activity actively monitored/targe
 online services (eg. Gmail, Google Play store) failing to login or being forced to solve endless capchas
 when accessing sites that use Cloudflare's CDN services.
 
+Fix DNS leak;
+1. Enable "Do not send UDP(allow only local)" checkbox.
```
Comment (M66B) [inline FAQ.md]: This will not be added to NetGuard

**Comment by M66B** [comment]
Invalid

**Pull request closed without merging**